routers/web/web.go

```diff
@@ -1368,6 +1368,7 @@ func registerRoutes(m *web.Route) {
 				m.Post("/approve", reqRepoActionsWriter, actions.Approve)
 				m.Post("/artifacts", actions.ArtifactsView)
 				m.Get("/artifacts/{artifact_name}", actions.ArtifactsDownloadView)
+				m.Delete("/artifacts/{artifact_name}", actions.ArtifactsDeleteView)
```

Comment:
Does it require `reqRepoActionsWriter` permission?

Reply:
Yes. The check is in the function `ArtifactsDeleteView` internal code but not here. It's better to move it out of that function.